data/json/items/comestibles/junkfood.json

```
@@ -1133,14 +1116,13 @@
     "weight": "40 g",
     "color": "brown",
     "spoils_in": "180 days",
-    "stim": 3,
     "container": "box_small",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": -2,
     "healthy": -2,
     "calories": 200,
-    "description": "Crunchy sugar in chocolate capsules.  Legal and stimmy.",
+    "description": "Crunchy sugar in chocolate capsules.  Legal and tasty.",
```

"Legal" sounds even weirder now than it did before. Here's my suggestion:
```suggestion
    "description": "Crunchy sugar in chocolate capsules.  The malt gives them a distinct, yet delicious flavor.",
```

---

I interpreted it as a joke on the use of the word "capsule", but you have a point. I've rebased in your suggestion.